src/neo-vm/ExecutionEngine.cs

```diff
@@ -58,6 +58,10 @@ public class ExecutionEngine : IDisposable
 
         public RandomAccessStack<ExecutionContext> InvocationStack { get; } = new RandomAccessStack<ExecutionContext>();
         public RandomAccessStack<StackItem> ResultStack { get; } = new RandomAccessStack<StackItem>();
+
+        //add a static Evaluation for storage infos for everyScript;
+        public RandomAccessStack<StackItem> StaticEvaluationStack { get; } = new RandomAccessStack<StackItem>();
```

Comment:
I think we should move it to `ExecutionContext`. It should be isolated between contracts.

Reply:
> I think we should move it to `ExecutionContext`. It should be isolated between contracts.

ExecutionContext is for function
but staticvars is for script

script A: has two function A1 A2
script B:has one function B1

we got 3 executionContext.  when InvocationStack is:A1()->B1()->A2().

Reply:
I don't think we need to consider this situation. If `A1()->B1()->A2()`, we can think of A1 and A2 as being in separate scripts. Because when `B1()->A2()`, it cannot call `A2()` directly, it must call the entry point (`Main`) of `A`. In the `Main` function, `A` will re-execute the codes for setting static members.

Reply:
What happens if A1() invokes A2() ?
Static values should be kept between calls, right?
Will it still happen when we put it on Context ?

Reply:
But Different calls have different context... so is not one for each script

Reply:
For `CALL`, we use `ExecutionContext.Clone()`.

https://github.com/neo-project/neo-vm/blob/73d23b11540e289bf11fdc49c54904e28c547a2e/src/neo-vm/ExecutionContext.cs#L75-L78

It reuse the stacks from the calling context.

Reply:
Yes, so if we put the stack inside, we need to copy it

Reply:
@erikzhang I agree.i will check Clone and find another implement way.